src/model/constants/DraftBlockType.js

```diff
@@ -27,7 +26,6 @@ export type DraftBlockType = (
   'unordered-list-item' |
   'ordered-list-item' |
   'blockquote' |
-  'pullquote' |
```

Comment:
Paragraph and pullquote were never used / referenced and were causing static type errors using flow. If any application requires it it can then use the configurable API to include it as demonstrated on the rich.html example file